vm/interpreter/src/gas_tracker/gas_charge.rs

```diff
@@ -22,4 +24,8 @@ impl GasCharge {
     pub fn total(&self) -> i64 {
         self.compute_gas + self.storage_gas
     }
+
+    pub fn for_charge(&self) -> (&'static str, Gas) {
+        (self.name, Gas::new(self.total()))
+    }
```

Comment:
Or you impl `From<GasCharge> for fvm::gas::GasCharge` and then use `apply_charge`.

Reply:
This will also make it easier to eventually get rid of `GasCharge` and just use `fvm::gas::GasCharge` directly.

Reply:
Done

Reply:
I guess this function can be deleted now.

Reply:
Done